src/components/QuestionPlay/WithoutOptions.tsx

```diff
@@ -48,15 +91,28 @@ export default function WithoutOptions({
         sx={(theme) => getQuestionTextStyles(theme, !!options[0].text && options[0].isCorrect)}>
         <SanitizedHtml>{options[0].text}</SanitizedHtml>
       </Box>
-      {!isAttempted && (
+      {userResponse.isConfirmed ? (
         <Group>
-          <Button color="red" onClick={() => handleSelectedChoice([])}>
-            Incorrect
+          <Button variant="outline" onClick={() => handleSelectedChoice(userResponse.value)}>
```

Comment:
if there are many subjective questions (or if the user is doing a quick testing for his own quiz), it would be frustrating for user to wait for 5 sec after each response

Reply:
They don't need to wait for 5s. We automatically save it after 5s. Even if they click another question. I think better to have the timer outside of the component in a helper function.

Reply:
I have a better idea. Show the continue button instead of this button. Clicking continue will save the response and stop timer, right?

Reply:
<img width="1387" alt="Screenshot 2023-10-20 at 1 17 30 PM" src="https://github.com/rajatkantinandi/quizmaster/assets/14038952/785bbb84-fac9-433e-b95d-b2d0f8c4fdf0">
<img width="1464" alt="Screenshot 2023-10-20 at 1 17 37 PM" src="https://github.com/rajatkantinandi/quizmaster/assets/14038952/3ee34377-ff1f-4af6-a883-a0ec3a8ac44a">

This is the current UI, user can't click another question before submitting response

Reply:
showing 'Continue' button instead of confirm Submit